From 4fe62e6b4c072676d246be02ba92a1df3a9d0feb Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Fri, 16 Mar 2012 01:34:10 +0100
Subject: [PATCH] Makefile: move binary and avoid empty dirs

Install to /usr/bin by default, and don't install an empty /etc.
---
 Makefile |   10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 566a848..c017509 100644
--- a/Makefile
+++ b/Makefile
@@ -6,12 +6,10 @@ CC := gcc
 all: bootchartd
 
 install: bootchartd
-	mkdir -p $(DESTDIR)/sbin
-	mkdir -p $(DESTDIR)/etc
-	mkdir -p $(DESTDIR)/usr/share/doc/bootchart/
-	install -m0755 bootchartd $(DESTDIR)/sbin/
-	install -m0644 bootchartd.conf.example $(DESTDIR)/usr/share/doc/bootchart/
-	install -m0644 README COPYING $(DESTDIR)/usr/share/doc/bootchart/
+	install -d $(DESTDIR)/usr/bin
+	install -d $(DESTDIR)/usr/share/doc/bootchart
+	install -m0755 bootchartd $(DESTDIR)/usr/bin/
+	install -m0644 README COPYING bootchartd.conf.example $(DESTDIR)/usr/share/doc/bootchart/
 
 OBJS := log.o svg.o bootchart.o
 
-- 
1.7.9.4

